client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -218,7 +218,7 @@ export class EditorMediaModalDetailItem extends Component {
 			return null;
 		}
 
-		return <EditorMediaModalDetailFields site={ site } item={ item } />;
+		return item && <EditorMediaModalDetailFields site={ site } item={ item } />;
```

Comment:
Is this the right place to check for `item`? Shouldn't it be moved above where the `! canUserUploadFiles` check is?